apps/create-fuels-counter-guide/test/ui/test-ui.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+pnpm run fuels:dev > /dev/null 2>&1 &
+sleep 5
+
+pnpm run original:dev > /dev/null 2>&1 &
+sleep 5
+
+pnpm exec playwright install --with-deps  > /dev/null 2>&1
+pnpm exec playwright test
```

Comment:
We may not need a script for this (may be able to refactor internally also), but have you checked out the [web server config](https://playwright.dev/docs/api/class-testconfig#test-config-web-server), we can launch multiple ahead of a test run. I assume it tears down also, may be better than a custom script.